tests/conftest.py

```diff
@@ -12,20 +12,24 @@
 logging.basicConfig(level=logging.DEBUG)
 
 
+@pytest.fixture(name="loop")
+async def loop_fixture():
+    """Return the event loop."""
+    return asyncio.get_running_loop()
+
+
 @pytest.fixture
-async def aioclient_mock(event_loop):
+async def aioclient_mock(loop):
     """Fixture to mock aioclient calls."""
-    loop = event_loop
     with mock_aiohttp_client(loop) as mock_session:
         yield mock_session
 
 
 @pytest.fixture
-async def cloud_mock(event_loop, aioclient_mock):
+async def cloud_mock(loop, aioclient_mock):
     """Yield a simple cloud mock."""
-    loop = event_loop
     cloud = MagicMock(name="Mock Cloud", is_logged_in=True)
-    cloud.run_task = loop.create_task
+    cloud.run_task = asyncio.create_task
```

Comment:
The `Cloud.run_task` property is still using `loop.create_task` should this change here instead of being consistent?

https://github.com/NabuCasa/hass-nabucasa/blob/376773fbfe94e8a06dc591189ea2a33cf28cc13b/hass_nabucasa/__init__.py#L212

Reply:
I suggest we remove the `run_task` method as it's not needed. Each place can just use `asyncio.create_task` instead. But we should do it separately.